prql-compiler/src/parser.rs

```diff
@@ -220,7 +220,15 @@ fn ast_of_parse_pair(pair: Pair<Rule>) -> Result<Option<Node>> {
                 pipeline: Box::new(pipeline),
             })
         }
-        Rule::ident | Rule::jinja => Item::Ident(pair.as_str().to_string()),
+        Rule::ident | Rule::jinja => {
+            let inner = pair.as_str();
+            let stripped = if inner.starts_with('`') && inner.ends_with('`') {
+                &inner[1..inner.len() - 1]
```

Comment:
String slicing like this can fail on unicode; we should instead use `.chars`

(Or add a Todo and I'm happy to take this)

Reply:
Oh, yeah that is a must fix before merging, thanks.